src/commands/force/user/password/generate.ts

```diff
@@ -45,13 +69,44 @@ export class UserPasswordGenerateCommand extends SfdxCommand {
         const connection: Connection = await Connection.create({ authInfo });
         const org = await Org.create({ connection });
         const user: User = await User.create({ org });
-        const password = User.generatePasswordUtf8();
+        let passwordCondition: PasswordConditions;
+        if (connection.metadata) {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          const list: any = await connection.metadata.list({ type: 'ProfilePasswordPolicy' });
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
+          const profPasswordPolicies: ProfilePasswordPolicy = JSON.parse(
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+            JSON.stringify(await connection.metadata.read('ProfilePasswordPolicy', [list.fullName]))
```

Comment:
@maggiben, is this ProfilePasswordPolicy associated with the profile the user is assigned?

Reply:
pretty sure that's wrong.  You'll need to match the ProfilePasswordPolicy with the name of the user's profile.  Which this command doesn't have yet (so that should probably be the first query that happens)